libraries/rapi_users.rb

```diff
@@ -6,10 +6,10 @@
 #   'write' => true # or false
 def rapi_users(h)
   rapi_users = []
-  h.reject {|k,v| k == 'id' }.each_pair do |username,opts|
-    md5 =  username + ":Ganeti Remote API:" + opts['password']
-    md5 = Digest::MD5.new().update(md5).hexdigest
-    rapi_users << "#{username} {HA1}#{md5} #{opts['write'] ? "write" : ""}"
+  h.reject { |k| k == 'id' }.each_pair do |username, opts|
+    md5 =  username + ':Ganeti Remote API:' + opts['password']
```

Comment:
This should be renamed to `rapi_user` or something more true to what it represents, and should use string substitution:
```ruby
rapi_user = "#{username}:Ganeti Remote API:#{opts['password']}"
md5 = Digest::MD5.new.update(rapi_user).hexdigest
```
Also, there is an extra space after the equal sign that should be removed.